src/main/java/com/rocketden/main/service/RoomService.java

```diff
@@ -110,6 +110,29 @@ public RoomDto getRoom(String roomId) {
         return RoomMapper.toDto(room);
     }
 
+    public RoomDto removeUser(String roomId, String userId) {
+        Room room = repository.findRoomByRoomId(roomId);
+
+        // Return error if room could not be found
+        if (room == null) {
+            throw new ApiException(RoomError.NOT_FOUND);
+        }
+
+        User user = room.getUserByUserId(userId);
```

Comment:
Add a check to make sure the initiator of the request is the host, otherwise through an invalid permissions error

+ associated tests in RoomTests and/or RoomSocketTests